scripts/ipr-check.js

```diff
@@ -154,11 +154,12 @@ const usernames = request(sheetData).then((json) => JSON.parse(json)).then(data
 });
 
 const exceptions = new Set([
-  'EricSL', // Google employee
+  'fitzgen', // former Mozilla employee
+  'lforst', // Sentry employee
```

Comment:
@lforst Could you ask @mitsuhiko to make you a TC39 delegate, since the IPR policy is that you either have to be a delegate or contribute as an individual / invited expert?

Reply:
How can I do that? :)

Reply:
Open an issue in https://github.com/tc39/Admin-and-Business/!